server/lib/routes.js

```diff
@@ -28,9 +28,6 @@ module.exports = function (config, i18n) {
     require('./routes/get-client.json')(i18n),
     require('./routes/get-config')(i18n),
     require('./routes/get-metrics-errors')(),
-    require('./routes/get-openid-authenticate')(config),
-    require('./routes/get-openid-configuration')(config),
```

Comment:
Let's keep the `/.well-known/openid-configuration` endpoint, that's for reliers that want to use us as an IdP, rather than for us accepting logins from other providers.